Cirumflex_Combined.ino

```diff
@@ -72,6 +84,10 @@ void setup() {
     PWM.begin(dLunitPins[led]);
   }
 
+  Serial.println("Pixel demo...");
+  //PWM.demo(); // show something predictable at startup
+  Serial.println("Pixel demo done");
```

Comment:
I was playing with the idea that: At startup, show the RGB values so you know you have the wiring/definitions correct. Only works for neo at the moment.